src/data.js

```diff
@@ -22,19 +22,25 @@ genotet.data = {};
  * Data queries are sent to this address via http and received via jsonp.
  * @type {string}
  */
-genotet.data.serverURL;
+genotet.data.serverUrl;
 
 /**
- * Data are uploaded to this URL by posting multipart form data.
+ * Data are uploaded to this Url by posting multipart form data.
  * @type {string}
  */
-genotet.data.uploadURL;
+genotet.data.uploadUrl;
 
 /**
  * Log queries are sent to this address.
  * @type {string}
  */
-genotet.data.logURL;
+genotet.data.logUrl;
+
+/**
+ * Url for queries for monitoring data processing.
+ * @type {string}
+ */
+genotet.data.lploadProgressUrl;
```

Comment:
Looks like error from global replace. Be careful. With WebStorm you can actually view the list of find results before replacing them.